### PR TITLE
Russian translation of job config page items

### PR DESCRIPTION
**Pull request by Riliane**
Translations of many items and options on the job config page, fixing of a couple misspellings, and uncorrupting one help button blurb corrupted by bad encoding.

### Proposed changelog entries

- Add translation of the job configuration page content into Russian

### Proposed upgrade guidelines

N/A

### Submitter checklist

- [N/A] The Jira issue, if it exists, is well-described.
- [x] The changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developers, depending on the change) and are in the imperative mood (see [examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)).
  - Fill in the **Proposed upgrade guidelines** section only if there are breaking changes or changes that may require extra steps from users during upgrade.
- [N/A] There is automated testing or an explanation as to why this change has no tests.
- [N/A] New public classes, fields, and methods are annotated with `@Restricted` or have `@since TODO` Javadocs, as appropriate.
- [N/A] New deprecations are annotated with `@Deprecated(since = "TODO")` or `@Deprecated(forRemoval = true, since = "TODO")`, if applicable.
- [N/A] New or substantially changed JavaScript is not defined inline and does not call `eval` to ease future introduction of Content Security Policy (CSP) directives (see [documentation](https://www.jenkins.io/doc/developer/security/csp/)).
- [N/A] For dependency updates, there are links to external changelogs and, if possible, full differentials.
- [N/A] For new APIs and extension points, there is a link to at least one consumer.

### Desired reviewers

@Pldi23 @yaroslavafenkin @duemir 

### Maintainer checklist

Before the changes are marked as `ready-for-merge`:

- [ ] There are at least two (2) approvals for the pull request and no outstanding requests for change.
- [ ] Conversations in the pull request are over, or it is explicit that a reviewer is not blocking the change.
- [ ] Changelog entries in the pull request title and/or **Proposed changelog entries** are accurate, human-readable, and in the imperative mood.
- [ ] Proper changelog labels are set so that the changelog can be generated automatically.
- [ ] If the change needs additional upgrade steps from users, the `upgrade-guide-needed` label is set and there is a **Proposed upgrade guidelines** section in the pull request title (see [example](https://github.com/jenkinsci/jenkins/pull/4387)).
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins.io/issues/?filter=12146)).

<a href="https://gitpod.io/#https://github.com/jenkinsci/jenkins/pull/7230"></a>

### CODE REVIEW

File: core/src/main/resources/hudson/model/AbstractProject/help-concurrentBuild_ru.html
Comment (daniel-beck) [inline core/src/main/resources/hudson/model/AbstractProject/help-concurrentBuild_ru.html:48]: Docs are no longer in the wiki; any such references should just be to "documentation" or maybe "website".

Comment (Riliane) [reply core/src/main/resources/hudson/model/AbstractProject/help-concurrentBuild_ru.html:48]: has to be fixed in the English version too then

File: core/src/main/resources/hudson/model/BooleanParameterDefinition/config_ru.properties
Comment (Pldi23) [inline core/src/main/resources/hudson/model/BooleanParameterDefinition/config_ru.properties:3]: I'm not sure, why did we need to add Cloudbees for license?

File: core/src/main/resources/hudson/tasks/Maven/config_ru.properties
Comment (Pldi23) [inline core/src/main/resources/hudson/tasks/Maven/config_ru.properties:31]: May be better: "Использовать переменные сборки" ?

File: core/src/main/resources/hudson/triggers/Messages_ru.properties
Comment (Pldi23) [inline core/src/main/resources/hudson/triggers/Messages_ru.properties:3]: SCMTrigger.no_schedules_no_hooks=Post-commit хуки игнорируются и не включаются в расписание к последующим запускам, т.е. никогда не будут запущены в SCM

Comment (Pldi23) [reply core/src/main/resources/hudson/triggers/Messages_ru.properties:3]: SCMTrigger.no_schedules_hooks=Не включаются в расписание последующих запусков, поэтому это может быть запущено только изменением в SCM \
  при помощи post-commit хука

File: core/src/main/resources/hudson/triggers/TimerTrigger/help_ru.html
Comment (Pldi23) [inline core/src/main/resources/hudson/triggers/TimerTrigger/help_ru.html:2]:
```suggestion
  Предоставляет <a href="https://en.wikipedia.org/wiki/cron">cron</a>-подобную функциональность для указания расписания
```

File: core/src/main/resources/hudson/triggers/TimerTrigger/help_ru.html
Comment (Pldi23) [inline core/src/main/resources/hudson/triggers/TimerTrigger/help_ru.html:19]: Может быть лучше: "Пожалуйста, перед тем как использовать эту функцию, убедитесь, что вы действительно хотите именно этого" ?

File: core/src/main/resources/jenkins/tasks/filters/impl/RetainVariablesLocalRule/config_ru.properties
Comment (Pldi23) [inline core/src/main/resources/jenkins/tasks/filters/impl/RetainVariablesLocalRule/config_ru.properties:24]:
```suggestion
  Заданные вне Jenkins переменные будут возвращены в значения по умолчанию или удалены, в зависимости от выбранной \
```

**Review by yaroslavafenkin — COMMENTED** [review]
Some terms sound so weird in russian and are probably more confusing than if they weren't translated. Or maybe it's just me :D

File: core/src/main/resources/hudson/triggers/Messages_ru.properties
Comment (Riliane) [reply core/src/main/resources/hudson/triggers/Messages_ru.properties:3]: там, по идее, не хуки не включаются в расписание, а ни хуков нет, ни расписания нет. это отдельные вещи - я так понимаю, можно запустить сразу после коммита хуком, а можно время от времени расписанием
Post commit hooks are being ignored and no schedules so will never run due to SCM changes

Comment (Pldi23) [reply core/src/main/resources/hudson/triggers/Messages_ru.properties:3]: Ah yes, I reread and looks you're right (It potentially means that english version slightly misleading).

File: core/src/main/resources/hudson/model/AbstractProject/help-concurrentBuild_ru.html
Comment (Riliane) [reply core/src/main/resources/hudson/model/AbstractProject/help-concurrentBuild_ru.html]: I fixed it in the English version too.

File: core/src/main/resources/hudson/tasks/Maven/config_ru.properties
Comment (Riliane) [reply core/src/main/resources/hudson/tasks/Maven/config_ru.properties:31]: Pass all build variables into maven process in form of java properties. This is seldom needed as Jenkins provides it as environment variables anyway. Preferred way to access Jenkins build variables is to explicitly map it to property in Properties section ( MY_VAR=${MY_VAR} ).

seems like there's multiple ways to _use_ the build variables and this is just one of them. If you can word it better from that summary that'd be nice, it does sound somewhat awkward 😔

File: core/src/main/resources/hudson/triggers/Messages_ru.properties
Comment (Riliane) [reply core/src/main/resources/hudson/triggers/Messages_ru.properties:3]: would this sentence sound less weird as maybe "Post-commit хуки игнорируются, запланированных запусков нет, поэтому сборка никогда не будет запущена изменением в SCM"?

**Comment by Riliane** [comment]
@yaroslavafenkin this is definitely a challenge, a lot of people discuss such technical things in halfway-English 😂

File: core/src/main/resources/hudson/triggers/Messages_ru.properties
Comment (Pldi23) [reply core/src/main/resources/hudson/triggers/Messages_ru.properties:3]: I would also think that we don't need to translate "hook". Would "Post-commit hooks игнорируются, запланированных запусков нет, поэтому сборка никогда не будет запущена изменением в SCM" sounds better?

File: core/src/main/resources/hudson/tasks/Maven/config_ru.properties
Comment (Pldi23) [reply core/src/main/resources/hudson/tasks/Maven/config_ru.properties:31]: Might be "Применить переменные сборки"?

File: core/src/main/resources/hudson/triggers/Messages_ru.properties
Comment (Pldi23) [reply core/src/main/resources/hudson/triggers/Messages_ru.properties:3]: ok, let's be consistent with git docs then

**Review by Pldi23 — APPROVED** [review]

File: core/src/main/resources/hudson/triggers/Messages_ru.properties
Comment (Riliane) [reply core/src/main/resources/hudson/triggers/Messages_ru.properties]: I occasionally try to google the options for some terms I have doubts about the translation of, and see the hits that are brought up, see if it is a word actually used in docs, guide articles, etc.

**Review by yaroslavafenkin — APPROVED** [review]

**Review by basil — APPROVED** [review]
Thanks for the PR! I was worried that the Mojibake in `core/src/main/resources/hudson/triggers/TimerTrigger/help_ru.html` was a more widespread problem and/or a recent regression, but (fortunately!) neither appears to be the case. This particular file appears to have been corrupted over a decade ago in commit 825dc6dbcf73f4c443d4e71f41fc10a3130686e9. I found and fixed a few other isolated cases like this in #7248, #7249, and #7250, all of which date back to the same time period. Hopefully the consistent use of UTF-8 encoding introduced in #6719 will make it harder for these kinds of errors to be introduced in the future.

---

This PR is now ready for merge. We will merge it after approximately 24 hours if there is no negative feedback. Please see the [merge process documentation](https://github.com/jenkinsci/jenkins/blob/master/docs/MAINTAINERS.adoc#merge-process) for more information about the merge process. Thanks!

**Review by duemir — APPROVED** [review]
I am a bit late, but here is my 🐝 as well :)
I checked thoroughly the first ten files the rest just skimmed since it all looks pretty good.

**Comment by Riliane** [comment]
> Thanks for the PR! I was worried that the Mojibake in `core/src/main/resources/hudson/triggers/TimerTrigger/help_ru.html` was a more widespread problem and/or a recent regression, but (fortunately!) neither appears to be the case. This particular file appears to have been corrupted over a decade ago in commit [825dc6d](https://github.com/jenkinsci/jenkins/commit/825dc6dbcf73f4c443d4e71f41fc10a3130686e9). I found and fixed a few other isolated cases like this in #7248, #7249, and #7250, all of which date back to the same time period. Hopefully the consistent use of UTF-8 encoding introduced in #6719 will make it harder for these kinds of errors to be introduced in the future.
> 
> This PR is now ready for merge. We will merge it after approximately 24 hours if there is no negative feedback. Please see the [merge process documentation](https://github.com/jenkinsci/jenkins/blob/master/docs/MAINTAINERS.adoc#merge-process) for more information about the merge process. Thanks!

It was corrupted even before that. The old version isn't readable Russian either 😄 It probably somehow slipped through in the first commit, then I guess someone's IDE tried to decode it and corrupted it worse. The new version was absolutely unrecoverable after getting baked twice, but I was able to put the old version into an automatic encoding switch tool and recover the initial content, then apply the changes from the next commits back to it.

**Comment by basil** [comment]
Between the archaeological work done in this PR, #7248, #7249, and #7250, I think we have eradicated mojibake from this repository. I ran the `ftfy.badness.is_bad` heuristic from [this package](https://ftfy.readthedocs.io/en/latest/heuristic.html) on every line of every text file (as reported by [`file(1)`](https://www.darwinsys.com/file/)) in this repository and could not find any remaining mojibake.

**Pull request merged**
